rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -117,6 +119,9 @@ protected void prePopulateRDTPatientFields(Patient patient, JSONObject field) th
         } else if (PATIENT_AGE.equals(key)) {
             field.put(JsonFormUtils.VALUE, patient.getAge());
         }
+        else if (CovidConstants.FormFields.FACILITY_NAME.equals(key) || CovidConstants.FormFields.HEALTH_FACILITY_NAME.equals(key) || CovidConstants.FormFields.OTHER_HEALTH_FACILITY_TYPE_NAME.equals(key)) {
```

Comment:
Any reason for the 3 checks? Are there multiple fields on the RDT form that require to be pre-populated with the locations or is it just the `CovidConstants.FormFields.FACILITY_NAME`.

Reply:
yeah, we need to implement this feature in 3 different forms because all three forms have a different field name. Therefore i have used to OR check.

Reply:
Ok, could we move the fields into a set and create a private method that checks if the key is part of that set? Similar to https://github.com/onaio/rdt-standard/blob/e71190b6976212242a686fbb9aa6e84a9ea4a7b0/rdt-open-reader/src/main/java/io/ona/rdt/util/RDTJsonFormUtils.java#L80.